R/utilities.R

```diff
@@ -4418,25 +4371,29 @@ classify_fl <- function(
 #' @param projection The projection of the samples. Only used to retrerive data through GAMBLR when it is not provided. Defaults to grch37.
 #' @param output The output to be returned after prediction is done. Can be one of predictoins, matrix, or both. Defaults to predictions.
 #' @param adjust_ploidy Whether to perform ploidy adjustment for the CNV data. Defaults to TRUE (recommended).
+#' @param include_N1 When running Lacy classifieer, indicate whether to set samples with NOTCH1 mutations to N1 group as described in Runge et al (2021). Defaults to FALSE.
 #'
 #' @return data frame with classification, binary matrix used in classification, or both
 #' @export
 #' @import data.table circlize tidyverse
 #'
 #' @examples
 #' test_meta <- get_gambl_metadata(case_set = "DLBCL-unembargoed")
-#' predictions <- classify_dlbcl_chapuy(these_samples_metadata = test_meta)
-#' matrix_and_predictions <- classify_dlbcl_chapuy(these_samples_metadata = test_meta, output = "both")
+#' predictions_chapuy <- classify_dlbcl(these_samples_metadata = test_meta)
+#' predictions_lacy <- classify_dlbcl(these_samples_metadata = test_meta, method = "lacy")
```

Comment:
How about calling it "hmrn" or "HMRN" as a fourth "model" that is really just using lacy + N1?

Reply:
Great idea! Changed to this and implemented this update